layouts/New/Blog.tsx

```diff
@@ -41,9 +44,17 @@ const BlogLayout: FC = async () => {
 
       <div className={styles.blogLayout}>
         <main>
-          <h1>{t('layouts.blog.title')}</h1>
-
-          <p>{t('layouts.blog.subtitle')}</p>
+          <header>
```

Comment:
This header should probably be extrated into its own component, and then unit tested

Reply:
let's do that 🚀